pio/ws2812/ws2812.c

```diff
@@ -12,6 +12,10 @@
 #include "hardware/clocks.h"
 #include "ws2812.pio.h"
 
+#define PIN_TX 2
```

Comment:
@kilograham Would it be worth using the new `PICO_DEFAULT_WS2812_PIN` here? Or is that specifically only for a single WS2812 rather than a string of WS2812s?

Reply:
I updated the code to use PICO_DEFAULT_WS2812_PIN. If it is not defined for board (e.g. the Pi Pico), a warning is generated and the pin 2 is used.